.github/workflows/gh-pages.yaml

```diff
@@ -48,6 +48,13 @@ jobs:
       - name: Install dependencies
         run: python3 -m pip install -r docs/requirements.txt
 
+      - name: Find and replace "%pip install cleanlab" in .ipynb files
+        uses: jacobtomlinson/gha-find-replace@v2
+        with:
+          find: "%pip install cleanlab"
+          replace: "%pip install git+https://github.com/${{ github.repository }}.git@${{ github.sha }}"
+          include: "**/*.ipynb"
```

Comment:
should only be doing this auto find/replace inside `docs/source` right?

Reply:
done!